CHANGES.rst

```diff
@@ -35,6 +35,8 @@ ramp_fitting
 
 - Switch multiprocessing method to ``fork_server``. [#249]
 
+- Refactor ramp fitting using a C extension to improve performance. [#156]
```

Comment:
stcal 1.7.0 got released 1-2 months ago, so this change log entry needs to move up to the unreleased section.